### ISSUE TITLE
package not found 404 @slack/client

### ISSUE THREAD

**Issue by GGYaX** [slackapi/node-slack-sdk#309]
npm install @slack/client failed. https://registry.npmjs.org/@slack/client returns 404.

[npm-debug.txt](https://github.com/slackapi/node-slack-sdk/files/692065/npm-debug.txt)

node version 7.2.1

Any idea ?

**Comment by Everesty** [comment]
npm should be requesting an escaped version of `@slack/client` (e.g. `"@slack%2Fclient"`), which [does return a valid result](https://registry.npmjs.org/@slack%2fclient) and your error log does indicate that npm is escaping the name correctly (log message 14) and sending the request to the right url (log message 20).

Further, I attempted running `npm install @slack/client` (I am also on node 7.2.1, and this was after a cache clear via `npm cache clean`) and it succeeded.

So, simply put, this is a you problem. I have absolutely no idea why this would happen outside of your network blocking the request to the registry.

**Comment by GGYaX** [comment]
Thanks, I will check.
